simulator/src/themer/customThemeAbstraction.js

```
@@ -3,42 +3,40 @@
-            color: themeOptions['--text-lite'],
-            description: 'text color',
-            ref: ['--text-lite', '--text-panel', '--text-dark'],
-        },
-        Borders: {
-            color: themeOptions['--br-secondary'],
-            description: 'borders color',
-            ref: ['--br-secondary'],
-        },
-    };
-};
+export default themeOptions => ({
```

ESLint

[error] Expected parentheses around arrow function argument.

(arrow-parens)